lib/pbench/server/api/resources/query_apis/datasets/datasets_mappings.py

```diff
@@ -115,4 +115,7 @@ def _get(self, params: ApiParams, request: Request) -> Response:
             return jsonify(result)
         except TemplateNotFound as e:
             self.logger.exception(str(e))
```

Comment:
While you are here, can you consider changing this to `self.logger.error(str(e))`?  Not sure why a stack trace is needed for catching a specific exception (maybe there is?).

Reply:
And consider applying that to other modules which have the same pattern?